site/civicrm.php

```diff
@@ -36,6 +36,11 @@ function civicrm_initialize() {
   require_once 'PEAR.php';
   $config = CRM_Core_Config::singleton();
 
+  // Set the time zone in both PHP and database
+  $joomlaUserTimezone = CRM_Core_Config::singleton()->userSystem->getTimeZoneOffset();
```

Comment:
@spalmstr please change it to
-  $joomlaUserTimezone = CRM_Core_Config::singleton()->userSystem->getTimeZoneOffset();
+  $joomlaUserTimezone = CRM_Core_Config::singleton()->userSystem->getTimeZoneString();